.github/workflows/automation-pr.yml

```
@@ -0,0 +1,19 @@
+# Automation that runs upon pull requests being submitted.
+name: automation-pr
+
+# Since pull requests from forks do not have repo write permission (for good
+# reason), we need to run on the `check_suite` trigger. This requires a non-
+# Github bot to be running on PRs to trigger the event.
+on:
+  check_suite:
+    types: [completed]
+
+jobs:
+  mirror-labels-to-pr:
+    runs-on: ubuntu-latest
+    container: quay.io/enarx/fedora-test
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: mirror-labels-to-pr
+      run: python3 .automation/mirror-labels-to-pr.py ${{ secrets.GITHUB_TOKEN }} $GITHUB_REPOSITORY $GITHUB_EVENT_PATH
```

```suggestion
      run: .automation/copy-issue-labels-to-pr ${{ secrets.GITHUB_TOKEN }} $GITHUB_REPOSITORY $GITHUB_EVENT_PATH
```

Rename the python script. Use a shebang line. `chmod 755 .automation/copy-issue-labels-to-pr`